scripts/vet.sh

```diff
@@ -183,4 +183,7 @@ revive -set_exit_status=1 -exclude "reflection/test/grpc_testing_not_regenerate/
 # TODO: Remove `|| true` to unskip linter failures once existing issues are fixed.
 (not grep -v "\.pb\.go:" "${REV_OUT}") || true
 
+# Ensure that no trailing spaces are found.
+not git grep '[[:blank:]]$'
```

Comment:
Optional: 

Should we group all the git grep together: near ~line:40

Reply:
yeah moved to line 70 after other not git grep